README.md

```diff
@@ -22,7 +22,7 @@ protractor-flake
 
 # Full options
 # protractor-flake <protractor-flake-options> -- <options to be passed to protractor>
-protractor-flake --protractor-path=/path/to/protractor --parser standard --node-bin node --max-attempts=3 -- path/to/protractor.conf.js
+protractor-flake --protractor-path=/path/to/protractor --parser standard --node-bin node --max-attempts=3 --color=magenta -- path/to/protractor.conf.js
```

Comment:
Can we have 'color' to be either a boolean or a color value. When it is boolean, we set a fixed color and when it is a color value we set the color to that value.

Some tools like mocha or jasmine provide a way for defining reporters. These reporters provide a way to customize what and how things get displayed on the screen. I have seen nyan cat reporters and airstrip reporters (https://mochajs.org/#reporters) which might not look practical but they support the idea of letting users decide what they want displayed on the screen :-)